reference_data/management/commands/utils/update_utils.py

```diff
@@ -85,11 +87,12 @@ def update_records(reference_data_handler, file_path=None):
     skip_counter = 0
     logger.info('Parsing file')
     open_file = gzip.open if file_path.endswith('.gz') else open
-    with open_file(file_path) as f:
+    open_mode = 'r' if file_path.endswith('.gz') else 'rb'
```

Comment:
For gzip file, open mode 'r' means binary mode while 'r' means text mode for other files. The files should be read in binary mode and decode later because Python 2 doesn't decode properly for the unicode data while reading a text file.
It is weird that `gzip` for Python 2 can decode unicode well while reading in text mode. To be aligned with other file types, the gzip files are also read in binary mode here.

Reply:
so none of these files are ever going to have unicode so we shouldn't make the code more complex to support it. For future reference, everything is already working in python 2 so you don't need to make changes to add additional python 2 support